docs/reference/pvc-finalizer-downgrade-issue.md

```diff
@@ -0,0 +1,32 @@
+* TOC
+{:toc}
+
+---
+title: Kubernetes Downgrade issue from 1.10 to 1.9 due to PV/PVC Protection
+---
+
+## PV/PVC Protection in Kubernetes 1.10
+
+When enabled, [PV/PVC Protection](docs/admin/admission-controllers.md#storage-object-in-use-protection-beta) prevents PV/PVC from being removed when the finalizers are removed. 
```

Comment:
"If you enable the admission controller `StorageObjectInUseProtection`, your PersistentVolume and PersistentVolumeClaim objects are not removed if the related `pv-protection` and `pvc-protection` finalizers are removed."

Not sure I've got this right because there seems to be some contradiction between the text ^^ and text below.

Reply:
good catch, it is should be `PersistentVolume and PersistentVolumeClaim objects are not removed if the related pv-protection and pvc-protection finalizers are still *present*`